Can't create graphs

After ran the program all day (`while true; do ./thyme track -o thyme.json; sleep 30s; done;`), I finished it (`Ctrl+C`) then tried to get the stats (`./thyme show -i thyme.json -w stats > thyme.html`), but this errors happened:
```
👉  ./thyme show -i thyme.json -w stats > thyme.html
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9dd96]

goroutine 1 [running]:
panic(0x1673e0, 0x10580008)
       	/home/beyang/go/src/runtime/panic.go:500 +0x325
github.com/sourcegraph/thyme.Stats.func1(0x0, 0x1071db00, 0xa)
       	/home/beyang/src/github.com/sourcegraph/thyme/show.go:17 +0x26
github.com/sourcegraph/thyme.NewTimeline(0x105663d0, 0x1a13c0, 0x73713)
       	/home/beyang/src/github.com/sourcegraph/thyme/show.go:142 +0x33c
github.com/sourcegraph/thyme.Stats(0x105663d0, 0x0, 0x0)
       	/home/beyang/src/github.com/sourcegraph/thyme/show.go:17 +0x32
main.(*ShowCmd).Execute(0x233e60, 0x1055e480, 0x0, 0x5, 0x0, 0x0)
       	/home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:114 +0x45b
github.com/jessevdk/go-flags.(*Parser).ParseArgs(0x1055e450, 0x105c6098, 0x5, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0)
       	/home/beyang/src/github.com/jessevdk/go-flags/parser.go:314 +0x8f3
github.com/jessevdk/go-flags.(*Parser).Parse(0x1055e450, 0x0, 0x0, 0x0, 0x0, 0x0)
       	/home/beyang/src/github.com/jessevdk/go-flags/parser.go:185 +0x72
main.main.func1(0x0, 0x0)
       	/home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:141 +0x27
main.main()
       	/home/beyang/src/github.com/sourcegraph/thyme/cmd/thyme/main.go:148 +0x1c
```

How should it be terminated? I wanted to see these stats :(

--- a/show.go
+++ b/show.go
@@ -163,7 +163,10 @@ func NewTimeline(stream *Stream, labelFunc func(*Window) string) *Timeline {
 		nextVisible := make(map[string]*Range)
 		for _, v := range snap.Visible {
 			win := windows[v]
-			winLabel := labelFunc(win)
+			var winLabel string
+			if win != nil {
+				winLabel = labelFunc(win)
+			}
 			if existRng, exists := lastVisible[winLabel]; !exists {
 				newRange := &Range{Label: winLabel, Start: snap.Time, End: snap.Time}
 				nextVisible[winLabel] = newRange
@@ -384,6 +387,9 @@ var statsTmpl = template.Must(template.New("").Funcs(map[string]interface{}{
 // window, w. It does so in best effort fashion. If the application
 // can't be determined, it returns the the name of the window.
 func appID(w *Window) string {
+	if w == nil {
+		return "(oops)"
+	}
 	if w.Info().App != "" {
 		return w.Info().App
 	}

